hack/make/test-docker-py

```diff
@@ -18,8 +18,7 @@ source hack/make/.integration-test-helpers
 # flag) until they are fixed upstream. For example:
 # --deselect=tests/integration/api_container_test.py::AttachContainerTest::test_attach_no_stream
 # TODO re-enable test_attach_no_stream after https://github.com/docker/docker-py/issues/2513 is resolved
-# TODO re-enable test_run_container_reading_socket_ws. It's reported in https://github.com/docker/docker-py/issues/1478, and we're getting that error in our tests.
```

Comment:
try re-enable AttachContainerTest::test_run_container_reading_socket_ws

temporary commit to see if this still fails on "main"